packages/app-store/paypal/api/add.ts

```diff
@@ -9,12 +9,18 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
     return res.status(401).json({ message: "You must be logged in to do this" });
   }
   const appType = config.type;
+  const teamId =
+    req.query?.teamId && typeof req.query?.teamId === "string"
+      ? parseInt(req.query?.teamId as string)
+      : undefined;
+  const whereClause =
+    teamId !== undefined && !Number.isNaN(teamId)
+      ? { type: appType, teamId }
+      : { type: appType, userId: req.session.user.id };
```

Comment:
You can simply it more. Also, we need to check if the user has admin access to team or not.